core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/EngineeringAllowanceManager.kt

```diff
@@ -84,7 +84,14 @@ class EngineeringAllowanceManager(private val graph: STDCMGraph) {
             }
         val routes = edges.last().infraExplorer.getExploredRoutes()
         val pathProperties = makePathProps(graph.rawInfra, graph.blockInfra, blockRanges, routes)
-        val mrsp = computeMRSP(pathProperties, graph.rollingStock, false, graph.tag)
+        val mrsp =
+            computeMRSP(
+                pathProperties,
+                graph.rollingStock,
+                false,
+                graph.tag,
+                null /* TODO add temporarySpeedLimitManager ? */
```

Comment:
```suggestion
                graph.temporarySpeedLimitManager,
```

This one would actually be relevant

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/9837/commits/3cad484c88edde0ebc00512b9f3cf3bc4e977b49